wongnung/tests/test_voting_view.py

```diff
@@ -71,3 +77,258 @@ def test_alternate_vote(self):
         self.client.post(url, {"down": "down"}, HTTP_HX_Request="true")
         self.assertEqual(self.review.get_upvotes(), 0)
         self.assertEqual(self.review.get_downvotes(), 1)
+
+
+class TestVotingE2E(StaticLiveServerTestCase):
+    """End-to-end tests for Voting view"""
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        options = ChromeOptions()
+        # options.add_argument("--headless")
```

Comment:
Posting a suggestion so we can commit automatically.
```suggestion
        options.add_argument("--headless")
```